virus/trojan_patchwork_virus.md

```diff
@@ -0,0 +1,107 @@
+
+## Abstract
+
+Research has suggested that some RNA-viruses have jumped from plant to insect genomes and vice versa (Kondo et al., 2019). Virga-like viruses are positive single-stranded RNA viruses that are closely related to the family Virgaviridae that is commonly associated with plants (Adams et al., 2009; Kondo et al., 2019). Recently, traces of a novel virga-like virus have been found in insects suggesting a common viral origin of these viruses between plants and insects. Here a novel virus (Trojan patchwork virus) with a virga-like virus RdRp was found in a sample of C. vestalis from a study by Zhao et al., 2017 from a search through RNA-sequencing databses. This novel virus acts as more evidence supporting a close evolutionary relationship between standard Virgaviridae plant viruses and virga-like viruses in insects and suggest that specific viruses may exist that can cross this transmission barrier. Further research should be conducted to focus on endogenous viral elements (EVEs) to see if more plant-like viruses are found in other insects, in the diamondback moth host that C. vestalis uses as a host, or in vegetation that C. vestalis interacts with.
+
+## Results
+
+### Ecology subsection A...
+
+The Trojan patchwork virus belongs to a virga-like family under *Virgaviridae* which is a family of positive-strand RNA viruses. These viruses are generally associated with plants but recent studies have found traces of this virus in insects suggesting that insects can act as a host and vector of viruses that infect plants (Kondo et al., 2019). 
+
+The sample (SRR3948414) containing this virus was part of a study conducted by Zhao et al., 2017 in which RNA-seq was conducted on two wasp species. This virus was found in *Costesia vestalis* which is a wasp that is a larval endoparasite of the diamondback moth. Doing BLASTP searches on related RdRp sequences showed that this virus was found in other related insects from similar geographical areas as *C. vestalis*. 
+
+Larval *C. vestalis* live inside or on the bodies of arthropods, including diamondback moths. They exist in the body cavity of the insect ultimately killing the host once the wasp emerges from its larval state. Diamondback moths are pests to brassicaceous crops and are not very receptive to insecticides. Therefore, *C. vestalis* have been used as biological control of these moths in many agricultural contexts in Japan (Hiroyoshi et al., 2017). 
+
+Phylogenetic analyses have shown that plant and insect virga-related viruses may share common viral origins (Kondo et al., 2019). It could be suggested that the Trojan patchwork virus could be closely related to a virga-like virus that originated in plants. It could also be possible for *C. vestalis* to pass this virus onto its host species such as diamondback moths. 
+
+### Virus Genome
+
+* Viruses in the Virgaviridae family are rod-shaped and 20-25 nm in diameter (Adams et al., 2012)
```

Comment:
Please do not include the "facts" list in the submitted Report. This was merely for your reseach. Either compile this information into paragraph form or remove it.